app/assets/javascripts/admin/templates/customize-themes-show.hbs

```diff
@@ -65,7 +65,7 @@
       {{/if}}
     {{/if}}
     {{#if model.remote_theme.about_url}}
-    <a class="url about-url" href="{{model.remote_theme.about_url}}">{{i18n "admin.customize.theme.about_theme"}} {{d-icon "link"}}</a>
+      <a class="url about-url" href="{{model.remote_theme.about_url}}">{{i18n "admin.customize.theme.about_theme"}} {{d-icon "link"}}</a>
```

Comment:
```suggestion
      <a class="url about-url" href="{{model.remote_theme.about_url}}">{{i18n "admin.customize.theme.about_theme"}}{{d-icon "link"}}</a>
```

Reply:
I don't think this one is correct
Before:
![Screenshot from 2019-11-27 11-41-44](https://user-images.githubusercontent.com/72780/69684013-01290780-110b-11ea-859b-1c882e04e137.png)
After:
![Screenshot from 2019-11-27 11-42-03](https://user-images.githubusercontent.com/72780/69684019-04bc8e80-110b-11ea-9bb5-fd570db7c41d.png)

Reply:
It might break your design but spacing shouldn’t be used for this. Spacing is for text not for html elements. You should use css for this IMO.

Reply:
got you, that's not my change, I only improved existing formatting, but you are right, let me fix it since I am working on that part